Dockerfile.exporter

```diff
@@ -2,11 +2,11 @@ ARG BUILDPLATFORM=linux/amd64
 FROM --platform=$BUILDPLATFORM debian:bullseye-slim as importer
 
 ARG PROMETHEUS_VARNISH_EXPORTER_VERSION=1.6.1
+RUN apt-get update && apt-get upgrade -y && apt-get install -y --no-install-recommends varnish curl tar gzip ca-certificates
 
-RUN apt-get update && apt-get upgrade -y && apt-get install -y --no-install-recommends varnish curl tar gzip ca-certificates \
-&& curl -L https://github.com/jonnenauha/prometheus_varnish_exporter/releases/download/${PROMETHEUS_VARNISH_EXPORTER_VERSION}/prometheus_varnish_exporter-${PROMETHEUS_VARNISH_EXPORTER_VERSION}.linux-amd64.tar.gz | \
-    tar -xz  && cp prometheus_varnish_exporter-${PROMETHEUS_VARNISH_EXPORTER_VERSION}.linux-amd64/prometheus_varnish_exporter /prometheus-varnish-exporter \
-    && chmod +x /prometheus-varnish-exporter
+RUN git clone https://github.com/jonnenauha/prometheus_varnish_exporter.git \
+  && cd prometheus_varnish_exporter \
+  && go build -o /prometheus-varnish-exporter
```

Comment:
Do you not need to explicitly set the GOOS and GOARCH here? Or is it smart enough to pull in the TARGETOS and TARGETARCH env variables?

Reply:
You are correct, you would need to pass those in if building cross platform (I was testing with multiple native builders).